conference.js

```diff
@@ -1834,7 +1834,7 @@ export default {
             APP.UI.showToolbar(6000);
         });
         room.on(JitsiConferenceEvents.SUBJECT_CHANGED,
-            subject => APP.API.notifySubjectChanged(subject));
+            subject => APP.store.dispatch(conferenceSubjectChanged(subject)));
```

Comment:
What would happen if you moved this into a middleware that web and native both use already?

Reply:
To move what exactly?

Reply:
The listener.

Reply:
Further explanation: There is concern about this logic being in conference.js on the web only. Is it an easy win to just move this into some existing or new middleware so that mobile redux also gets updated? I get the argument of "what about the rest of them" but what about this one?